src/library.js

```diff
@@ -3139,7 +3139,29 @@ LibraryManager.library = {
     // part of thier signature, so we rely the dynCall functions generated by
     // wasm-emscripten-finalize
     if (sig.includes('j')) {
+#if MAIN_MODULE
+      if(Module["dynCall_" + sig]){
+        return dynCallLegacy(sig, ptr, args);
```

Comment:
Hmm, reading this I'm confused as to why `dynCall_*` would only sometimes exist when `DYNCALLS`. That is, we should have returned from line 3135 above, I think? Is this related to this issue from the description:

> "Sometimes side modules need dyncalls with 64 bit types that are missing the dynCall wrappers from the main module."

If so, could we include dyncalls in side modules? cc @sbc100

Reply:
I will try to come up with a MWE soon. It occurred on this branch:
https://github.com/pyodide/pyodide/pull/2378
with the signature `viij` but I have seen it before.

> could we include dyncalls in side modules? cc @sbc100

Yeah I suspect you can do that, perhaps it's a better solution? `BigInt` support is much broader than `BigInt64Array` so I don't mind this. But yeah it would be harder for me to come up with this.